controller/inviteController.js

```diff
@@ -0,0 +1,121 @@
+const Invite = require('../model/inviteModel');
+const HttpStatus = require('http-status-codes');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const user = req.decoded;
+
+    //Only admins can create invite objects
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
+        return res.status(HttpStatus.FORBIDDEN).json({
+            status: StatusEnum['ERROR'],
+            message: 'Forbidden.'
+        })
+    }
+
+    const invite = new Invite(req.body);
+
+    invite.save().then(() => {
+        return res.status(HttpStatus.CREATED).json(invite);
```

Comment:
This is an error that I think all of us have. Calling `save` on an object instance will not set the `_id` property, could prove tedious if the id is needed straight after creation. Maybe you could do some research how to get it? Perhaps `then` takes the new invite as argument, but is omited since you already have a version of it (without id) declared above? Again, this is something that all of us have, so if you figure out how, share it with the rest of us so we can integrate changes as well.

Reply:
@zirlman has checked and you can do it as described in the above comment, so make those changes as well.